hazelcast-sql/src/main/java/com/hazelcast/sql/impl/schema/view/View.java

```diff
@@ -82,6 +88,7 @@ public void writeData(ObjectDataOutput out) throws IOException {
         }
         SerializationUtil.writeList(viewColumnNames, out);
         SerializationUtil.writeList(viewColumnTypes, out);
+        out.writeBoolean(streaming);
```

Comment:
Do we need a version guard here actually? E.g. `in.getVersion().isGreaterOrEqual(Versions.V5_3)`.. And it will most likely be a good idea to test this in a rolling upgrade setup just to be sure I guess (probably enough to just create some schema objects in 5.2 node and join a 5.3, verify that it has the schema objects and then drop the 5.2 member and verify that 5.3's schema is working as expected).

In a rolling upgrade these schema objects will still be transferred because they're in a regular IMap. We might need some mechanism post 5.2 -> 5.3 upgrade for handling that.

Reply:
Actually this field is no longer used if I initialize ViewTable's `streaming` property based on `OptUtils.isUnbounded`. I will remove it.

Reply:
bf9c2cc6ec4e4be6414e049dac0cc5ef419a8a5a

Reply:
the field was not removed from serialized form

Reply:
#24406